pkg/kubelet/kubelet_test.go

```diff
@@ -2666,6 +2728,21 @@ func TestUpdateNodeStatusWithoutContainerRuntime(t *testing.T) {
 	}
 	mockCadvisor.On("VersionInfo").Return(versionInfo, nil)
 
+	// Create a new DiskSpaceManager with a new policy. This new manager along with the mock FsInfo
+	// values added to Cadvisor should make the kubelet report that it is out of disk space.
```

Comment:
Thanks for catching that. It's a copy-paste fail. It is probably also an indication that this should be extracted out to a function. I will leave it as is for now. I will extract it out to a function if I make any more modifications.